packages/web-component/wordpress-plugin/README.md

```diff
@@ -0,0 +1,62 @@
+# Read Along WordPress Deployment Guide
+
+This page contains instructions on how to embed a read-along book in your WordPress site. In this guide, the assumptions are you have:
+
+- Basic understanding and ability to edit XML/HTML
+- Generated read-along either via the **cli** or exported a **web-bundle** from our **studio web app** (https://readalong-studio.mothertongues.org/).
+- Default installation of WordPress with block or classic editor
+- Familiarity with HTML editing on WordPress
+
+## Requirements
+
+- Administrator account to your WordPress site
+- (optional) Text editor ideally XML/HTML editor with syntax highlighting, e.g.VSCode, NotePad++
+
+## Installing the Read Along Web App Loader WordPress Plugin (Do once)
+
+You must have an administrator account on your WordPress site to install this plugin.
+
+### Manual Installation
+
+![Manual Installation Screenshot](https://raw.githubusercontent.com/ReadAlongs/Studio-Web/main/packages/web-component/wordpress-plugin/wp-read-along-web-app-loader/assets/screenshot-manual-install.png)
```

Comment:
I assume this link will work once we merge this PR.

Reply:
But you might be able to make it relative, like
```suggestion
![Manual Installation Screenshot](read-along-web-app-loader/assets/screenshot-manual-install.png)
```
Not required, but it would be nice not to have to worry about changing this readme if we move the repo or whatever. Same comment applies to all other links on this page.

Reply:
Incidentally there are two typos here: the prefix `wp-` was still in the path name, and the actual .png file in this PR is called `screenshort` instead of `screenshot`.